tests/virt/node/workload_density/test_kernel_samepage_merging.py

```
@@ -194,8 +194,8 @@ def test_pages_to_scan_grows_when_ksm_active(self, worker_node1, workers_utility
     @pytest.mark.rwx_default_storage
     @pytest.mark.polarion("CNV-10523")
     @pytest.mark.dependency(depends=["test_ksm_activated_when_node_under_pressure"])
-    def test_migrate_vm_when_ksm_active(self, ksm_label_added_to_worker2, vms_for_ksm_test):
-        migrate_vm_and_verify(vm=vms_for_ksm_test[0])
+    def test_migrate_vm_when_ksm_active(self, admin_client, ksm_label_added_to_worker2, vms_for_ksm_test):
```

Ruff (0.15.14)

[warning] Unused method argument: `ksm_label_added_to_worker2`

(ARG002)